### PR TITLE
Login styling tl

### PR DESCRIPTION
**Pull request by trishalanglois**
#### What's this PR do?
Styles Login Component (finally :))

#### Any background context you want to provide?
Just threw something together that uses many of our variables and mixins that we already had set up, so it looks really consistent with the rest of the app.  Down to change things, if either of you have suggestions, though!

#### Screenshots (if appropriate): 
Screen Shot 2020-01-04 at 9 58 37 PM

### CODE REVIEW

**Comment by Garrett-Iannuzzi** [comment]
I think it looks great. What do we want to do with the "Rancid" title?

File: src/Components/Login/Login.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@import '../../scss/variables';
+@import '../../scss/mixins';
+
+form {
```
Comment (Garrett-Iannuzzi) [inline src/Components/Login/Login.scss:4]: Should we nest these with scss?

**Review by Garrett-Iannuzzi — APPROVED** [review]

**Review by sertmer — COMMENTED** [review]
I think it looks great! We do have a lot of dead space around that login form, so I might suggest making them both wider and taller _or_ give them a background card to help fill some space. Even with a background I might make the inputs larger.

**Comment by trishalanglois** [comment]
> I think it looks great. What do we want to do with the "Rancid" title?

Ertie's on it!  It's part of the <Header /> component, so we talked about making Rancid bigger and the login button aligned on the right.

File: src/Components/Login/Login.scss (hunk 1 above)
Comment (trishalanglois) [reply src/Components/Login/Login.scss:4]: Do you mean the imports?  Not sure what you mean here.

Comment (Garrett-Iannuzzi) [reply src/Components/Login/Login.scss:4]: In regards to each css class. It's not a big deal but could make it a little easier to read.

Something like this... 

Screen Shot 2020-01-05 at 12 06 19 PM

**Pull request merged**